core/common/src/main/java/alluxio/AlluxioURI.java

```diff
@@ -492,7 +492,7 @@ public String toString() {
       String path = mUri.getPath();
       if (path.indexOf('/') == 0 && hasWindowsDrive(path, true) && // has windows drive
           mUri.getScheme() == null && // but no scheme
-          mUri.getAuthority() == null) { // or authority
+          mUri.getAuthority() instanceof NoAuthority) { // or authority
```

Comment:
shall we keep null check too>?
```suggestion
          (mUri.getAuthority() == null || mUri.getAuthority() instanceof NoAuthority)) { // or authority
```